src/pages/steps/AdditionalInformationStep/OfferScore.tsx

```diff
@@ -211,9 +218,27 @@ const OfferScore = ({ completedFields, offerId, scope, ...props }: Props) => {
 
     const { fieldName } = highestUncompletedValue;
 
-    return t(`create.additionalInformation.event_score.tip.${fieldName}`);
+    return fieldName;
   }, [fullCompletedFields, score, t]);
 
+  const TipLink = ({ field }: { field: string }) => {
+    const hash = field === 'video' ? 'media' : field;
+    return (
+      <Link
+        color={getValue('link')}
+        href={`#${hash}`}
+        onClick={(e) => {
+          e.preventDefault();
+          router.push({ hash }, undefined, {
+            shallow: true,
+          });
+        }}
```

Comment:
Is the `href` needed if it's overriden and prevented, or is it purely for ESLint purposes?

Reply:
Yes purely for esLint purpose